dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -421,6 +431,46 @@ public Response checkElasticSearch() {
         return Response.ok().build();
     }
 
+    @GET
+    @Timed
+    @Path("/health")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Successful response if the health checks succeed", description = "Successful response if the health checks succeed, NO authentication")
+    @ApiOperation(value = "Successful response if the health checks succeed", notes = "NO authentication", response = List.class)
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "All health checks successful", content = @Content(mediaType = MediaType.APPLICATION_JSON, array = @ArraySchema(schema = @Schema(implementation = HealthCheckResult.class))))
+    @ApiResponse(responseCode = HttpStatus.SC_BAD_REQUEST + "", description = "Bad Request")
+    @ApiResponse(responseCode = HttpStatus.SC_INTERNAL_SERVER_ERROR + "", description = "Health checks failed")
+    public Set<HealthCheckResult> checkHealth(
+            @Parameter(name = "include", description = "List of health checks to run. If unspecified, run all health checks", in = ParameterIn.QUERY,
+                    array = @ArraySchema(schema = @Schema(implementation = String.class, allowableValues = {"hibernate", "deadlocks", "connectionPool"})))
+            @ApiParam(name = "include", value = "List of health checks to run. If unspecified, run all health checks", allowableValues = "hibernate,deadlocks,connectionPool",
+                    type = "array") @QueryParam(value = "include") List<String> include) {
+        Map<String, HealthCheck.Result> results;
+        boolean allHealthy;
+        if (include.isEmpty()) { // Run all health checks
```

Comment:
More of an FY, but I was curious if `include` could ever be  `null`.  I couldn't get it to be. But I found out trying it that if you pass an `include` parameter with no value, e.g., `curl "localhost:8080/metadata/health?include"` -- you get a one item list, where the item's value is an empty string, which fails with `Could not run health checks. The following health checks don't exist: ''`.

This is effectively only for internal usage anyway, so no need to fix.